virtual_rainforest/core/data.py

```diff
@@ -0,0 +1,686 @@
+"""API documentation for the :mod:`core.data` module.
+************************************************** # noqa: D205
```

Comment:
At the moment this part of the module docstrings displays weirdly in the docs

Reply:
Damnit. `sphinx` doesn't like the `noqa` and if I leave it out `flake8` then fails.